src/main/java/smile/identity/core/Utilities.java

```diff
@@ -2,50 +2,57 @@
 
 // json converter;
 
-import org.json.simple.JSONObject;
+import org.apache.http.HttpResponse;
+import org.apache.http.client.HttpClient;
+import org.apache.http.client.methods.HttpGet;
+import org.apache.http.client.methods.HttpPost;
+import org.apache.http.entity.StringEntity;
+import org.apache.http.impl.client.DefaultHttpClient;
+import org.apache.http.util.TextUtils;
 import org.json.simple.JSONArray;
+import org.json.simple.JSONObject;
 import org.json.simple.parser.JSONParser;
 
-// apache http client
 import java.io.BufferedReader;
 import java.io.InputStreamReader;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
 
 // apache http client
-import org.apache.http.HttpResponse;
-import org.apache.http.HttpResponse;
-import org.apache.http.client.HttpClient;
-import org.apache.http.entity.StringEntity;
-import org.apache.http.client.methods.HttpPost;
-import org.apache.http.impl.client.DefaultHttpClient;
+// apache http client
 
 public class Utilities {
 
     private String partner_id;
     private String api_key;
     private String url;
-    private String sec_key;
-    private long timestamp;
 
+    @Deprecated
     public Utilities(String partner_id, String api_key, Integer sid_server) {
+        this(partner_id, api_key, String.valueOf(sid_server));
+    }
+
+    public Utilities(String partner_id, String api_key, String sid_server) {
         try {
-            this.partner_id = partner_id.toString();
+            this.partner_id = partner_id;
             this.api_key = api_key;
 
-            if (sid_server == 0) {
+            if (sid_server.equals("0")) {
                 url = "https://3eydmgh10d.execute-api.us-west-2.amazonaws.com/test";
-            } else if (sid_server == 1) {
+            } else if (sid_server.equals("1")) {
                 url = "https://la7am6gdm8.execute-api.us-west-2.amazonaws.com/prod";
+            } else {
+                this.url = sid_server;
```

Comment:
What is the purpose of allowing setting of `sid_server` other than TEST or prod?